src/main/java/ru/l3r8y/checks/ClassCanSuppress.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2023. Ivanchuck Ivan.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package ru.l3r8y.checks;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.cactoos.Scalar;
+import org.cactoos.list.ListOf;
+
+/**
+ * Check names that can be suppressed on class-level.
+ *
+ * @since 0.3.7
+ */
+@RequiredArgsConstructor
+public final class ClassCanSuppress implements Scalar<List<String>> {
+
+    @Override
+    public List<String> value() {
+        return new ListOf<>(
+            ErSuffixCheck.class.getSimpleName(),
```

Comment:
I think I need to move the puzzle here, it still requires manually adding each check

Reply:
@l3r8yJ 
I implement it a few days before, and it doesn't work on maven build, since packages are package getting erased in tests, works locally, but when we pull it on `maven install` it can scan it

Reply:
in this case puzzle still not solved, can you bring it back?

Reply:
@l3r8yJ I thought we can solve that by adding checks to `ClassCanSuppress.java`, I don't see any other ways to automate it

Reply:
@h1alexbel i think it's possible, can you put back #128 puzzle?

Reply:
@l3r8yJ OK, I can put it back, can you explain how you want to do it?
And I will try to implement it here in this pull request

Reply:
@l3r8yJ I've put it back